api/Dockerfile

```diff
@@ -9,17 +9,17 @@ RUN apt-get update && \
                     libpq-dev \
                     gettext
 
-RUN pip install pip -U
-
-WORKDIR /code
-COPY . /code
-
-RUN pip install -r requirements.txt
+COPY requirements.txt /requirements.txt
+RUN pip install -r /requirements.txt
 
 # The dev requirements should be installed only if a DEVBUILD variable is
 # passed to the docker build command. This should allow easy building of
 # the production container image.
-RUN if [ "x$DEVBUILD" != "x" ]; then pip install -r requirements-dev.txt; fi
+COPY requirements-dev.txt /requirements-dev.txt
+RUN if [ "x$DEVBUILD" != "x" ]; then pip install -r /requirements-dev.txt; fi
+
+WORKDIR /code
+COPY . /code
```

Comment:
Fixed some cache busting here. The image was building the above layers every time.